src/ci/bin/test.inc.sh

```diff
@@ -138,6 +144,7 @@ cromwell::private::create_build_variables() {
             CROMWELL_BUILD_MYSQL_PASSWORD="${CROMWELL_BUILD_MYSQL_PASSWORD-}"
             CROMWELL_BUILD_MYSQL_SCHEMA="${CROMWELL_BUILD_MYSQL_SCHEMA-cromwell_test}"
             CROMWELL_BUILD_GENERATE_COVERAGE=true
+            CROMWELL_RUN_TESTS=true
```

Comment:
could you DRY out everything by defaulting `CROMWELL_RUN_TESTS=true` and only setting it to `false` under the right circumstances?